Adapters/Vungle/ISVungleAdapter/ISVungleAdapter.m

```diff
@@ -1260,19 +623,18 @@ - (ISLoadWhileShowSupportState) getLWSSupportState:(ISAdapterConfig *)adapterCon
 #pragma mark - Legal Methods
 
 - (void)setConsent:(BOOL)consent {
-    LogAdapterApi_Internal(@"consent = %@", consent ? @"VungleConsentAccepted" : @"VungleConsentDenied");
-    [[VungleSDK sharedSDK] updateConsentStatus:(consent ? VungleConsentAccepted : VungleConsentDenied)
-                         consentMessageVersion:@""];
+    LogAdapterApi_Internal(@"consent = %@", consent ? @"Opt in" : @"Opt out");
+    [VunglePrivacySettings setGDPRStatus:consent];
+    [VunglePrivacySettings setGDPRMessageVersion:@""];
 }
 
-- (void) setCCPAValue:(BOOL)value {
+- (void)setCCPAValue:(BOOL)value {
     // The Vungle CCPA API expects an indication if the user opts in to targeted advertising.
     // Given that this is opposite to the ironSource Mediation CCPA flag of do_not_sell
     // we will use the opposite value of what is passed to this method
     BOOL optIn = !value;
-    VungleCCPAStatus status = optIn ? VungleCCPAAccepted : VungleCCPADenied;
-    LogAdapterApi_Internal(@"key = VungleCCPAStatus, value  = %@", optIn ? @"VungleCCPAAccepted" : @"VungleCCPADenied");
-    [[VungleSDK sharedSDK] updateCCPAStatus:status];
+    LogAdapterApi_Internal(@"key = VungleCCPAStatus, value  = %@", optIn ? @"Opt in" : @"Opt out");
+    [VunglePrivacySettings setCCPAStatus:optIn];
 }
 
 - (void)setMetaDataWithKey:(NSString *)key
```

Comment:
For cleanup, you can remove the stuff about ad orientation here. It's not supported anymore in v7

Reply:
@stanley-vungle Thanks a lot for the guidance, removed. ：）